.github/ISSUE_TEMPLATE/new_chain.yml

```
@@ -20,6 +20,13 @@ body:
       description: RPC URL
     validations:
       required: true
+  - type: checkboxes
+    id: allow_bacth_request
+    attributes:
+      label: The RPC endpoint must allow batch requests for the bot script.
+      options:
+        - label: I confirm that the RPC allows batch requests
+          required: true
```

```suggestion
  - type: checkboxes
    id: allow_batch_requests
    attributes:
      label: The RPC endpoint must allow batch requests for the refund estimation script.
      options:
        - label: I confirm that the RPC allows batch requests
          required: true
```
Minot improvement and typo fix